### PR TITLE
DHFPROD-5742: Add e2e custom roles test

### PR DESCRIPTION
**Pull request by ryanjdew**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress.json [hunk 1]
```diff
@@ -3,12 +3,17 @@
     "video": true,
     "viewportWidth": 1680,
     "viewportHeight": 1050,
+    "numTestsKeptInMemory": 1,
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/e2e/cypress.json:6]: I don't think this change is needed. e2e test failures have not been due to high memory use.

File: marklogic-data-hub-central/ui/e2e/cypress.json [hunk 2]
```diff
@@ -3,12 +3,17 @@
...
     "defaultCommandTimeout": 10000,
     "reporter": "junit",
     "reporterOptions": {
       "mochaFile": "results/cypress-report[hash].xml",
       "toConsole": true
     },
+    "retries": {
+      "runMode": 2,
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/e2e/cypress.json:14]: I don't think we need this. The tests shouldn't fail if everything is reliable.

**Review by brucean52 — CHANGES_REQUESTED** [review]
There are failures on the pipeline due to this PR.

**Review by brucean52 — APPROVED** [review]

**Review by xnikhil08 — APPROVED** [review]

File: marklogic-data-hub-central/ui/e2e/cypress.json (hunk 2 above)
Comment (ryanjdew) [reply marklogic-data-hub-central/ui/e2e/cypress.json:14]: I'll back out the retries configuration changes. 

Any strong feelings on the memory settings? 50 seems like a high default number and with the server running on the localhost if we add enough tests we could start running into resource contention issues.

Comment (brucean52) [reply marklogic-data-hub-central/ui/e2e/cypress.json:14]: If it's individual e2e tests, we have already exceeded 50 and I don't think adjusting the number will make a difference.

**Pull request merged**